<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainDialog</string>
  </property>
  <widget class="QStackedWidget" name="sw_page">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1001</width>
     <height>651</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page_home">
    <widget class="QLabel" name="lb_homeShow">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1000</width>
       <height>650</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resource.qrc">:/image/home.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_quickStart">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>550</y>
       <width>141</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_game">
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>-2</x>
       <y>0</y>
       <width>1001</width>
       <height>629</height>
      </rect>
     </property>
     <widget class="QPushButton" name="pb_test">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>测试按钮</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>400</y>
        <width>1001</width>
        <height>56</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>1</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>188</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="wdg_callLord" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pb_callLord">
            <property name="minimumSize">
             <size>
              <width>91</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>91</width>
              <height>32</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>幼圆</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-image: url(:/image/button.png);
color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>叫地主</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_noCall">
            <property name="minimumSize">
             <size>
              <width>91</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>91</width>
              <height>32</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>幼圆</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-image: url(:/image/button.png);
color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>不叫</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="wdg_playCards" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="pb_playCard">
            <property name="minimumSize">
             <size>
              <width>91</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>91</width>
              <height>32</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>幼圆</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-image: url(:/image/button.png);
color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>出牌</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_helpPlayCard">
            <property name="minimumSize">
             <size>
              <width>91</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>91</width>
              <height>32</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>幼圆</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-image: url(:/image/button.png);
color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>提示</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_pass">
            <property name="minimumSize">
             <size>
              <width>91</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>91</width>
              <height>32</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>幼圆</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-image: url(:/image/button.png);
color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>不出</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>157</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="lb_leftPass">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>180</y>
        <width>100</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resource.qrc">:/image/pass.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lb_rightPass">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>180</y>
        <width>100</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resource.qrc">:/image/pass.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lb_midPass">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>320</y>
        <width>100</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resource.qrc">:/image/pass.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_end">
    <widget class="QLabel" name="lb_resultShow">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1000</width>
       <height>650</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resource.qrc">:/image/win.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="lb_win">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>280</y>
       <width>191</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>幼圆</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>恭喜
 你赢了！</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_goHome">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>450</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_next_2">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>450</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
